storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -137,6 +137,14 @@
     @IsImplementationOfClass(implementsClass = IBlacklistStrategy.class)
     public static final String BLACKLIST_SCHEDULER_STRATEGY = "blacklist.scheduler.strategy";
 
+    /**
+     * Whether the {@link org.apache.storm.scheduler.blacklist.BlacklistScheduler} will put a
+     * (supervisor:slots) map to badSupervisorsToleranceSlidingWindow
```

Comment:
This will be too concrete about the implementations  and not easy for users to understand.  Maybe something like:

Whether {@link org.apache.storm.scheduler.blacklist.BlacklistScheduler}  will assume the supervisor is bad based on bad slots or not.  When some slot is bad, meaning the nimbus doesn't receive heartbeat from the worker in time, it's hard to determine if it's because of the supervisor node or the worker itself having issues.  If this is set to true, the scheduler will simply assume the supervisor is bad if there is any bad slot; otherwise, the scheduler will simply assume it's not a supervisor problem.

Reply:
Appreciate the suggestions. All make sense to me. Please check the latest changes.